<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNetSetEth0</class>
 <widget class="QDialog" name="DialogNetSetEth0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.505682 rgba(35, 97, 222, 247), stop:0.539773 rgba(61, 114, 223, 247), stop:1 rgba(93, 142, 239, 238));
	color: rgb(255, 255, 255);
	font: 16px;
}
QLabel { font:bold 22px;	color: rgb(255, 255, 255);}
QRadioButton {font:bold 22px;	color: rgb(255, 255, 255); }
QGroupBox::title {
  subcontrol-origin: margin;
  subcontrol-position: top center;
  padding-top: 10px;  
  border-top: 1px solid rgb(255, 255, 255);
  font:bold 24pt;
  color: rgb(255, 255, 255);
}
QLineEdit {
font: bold 32px; 
background-color: rgb(255, 255, 255);
border-radius: 4px;
border: 1px solid rgba(39, 22, 22, 254);
}
QGroupBox {
font: 75 18px;
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>有线网络配置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>字网掩码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>网   关</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MyIP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MyMask">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MyGateway">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_save_netconfig">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_cancel_netconfig">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_auto">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>280</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
   <property name="text">
    <string>自动获取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
